zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -673,37 +673,37 @@ def get_postgresql_databases(cluster_id, alias, environment, region, infrastruct
                                           sslmode='require')
         for db in dbnames:
             yield {
-                'id': '{}-{}'.format(db, pgcluster['id']),
+                'id': '{}-{}'.format(db, pgcluster.get('id')),
                 'type': POSTGRESQL_DATABASE_TYPE,
                 'kube_cluster': cluster_id,
                 'alias': alias,
                 'environment': environment,
                 'created_by': AGENT_TYPE,
                 'infrastructure_account': infrastructure_account,
                 'region': region,
-                'version': pgcluster['version'],
-                'postgresql_cluster': pgcluster['id'],
+                'version': pgcluster.get('version'),
```

Comment:
I think these are safe - unless there are lurking entities without the keys

Reply:
Better safe ;)

Reply:
absolutely :)